src/ztc/api/serializers/besluittype.py

```diff
@@ -1,21 +1,24 @@
+from django.utils.translation import ugettext_lazy as _
+
 from rest_framework import serializers
+from rest_framework.exceptions import PermissionDenied
 
-from ...datamodel.models import BesluitType
+from ...datamodel.models import BesluitType, InformatieObjectType, ZaakType
 
 
 class BesluitTypeSerializer(serializers.HyperlinkedModelSerializer):
     informatieobjecttypes = serializers.HyperlinkedRelatedField(
         view_name='informatieobjecttype-detail',
         many=True,
         lookup_field='uuid',
-        read_only=True
+        queryset=InformatieObjectType.objects.all()
```

Comment:
I think these need to get restricted to the same catalogue as the catalogue where the `BesluitInformatieObject` lives. Probably best to implement this as a validator?

Reply:
validator `RelationCatalogValidator` is added to `BesluitType` and `ZaaktTpe` serializers